cmd/util/util.go

```diff
@@ -34,10 +34,12 @@ import (
 // writeIfNotFound returns true if the file was created and false if it already exists
 func WriteIfNotFound(path, templateName, templateValue string, data interface{}) bool {
 	// Make sure the directory exists
-	os.MkdirAll(filepath.Dir(path), 0700)
-
+	err := os.MkdirAll(filepath.Dir(path), 0700)
+	if err != nil {
+		return false
```

Comment:
The description of the function says that it returns `false` if it already exists and here it is returning false if we are unable to make the directories. I would suggest using an `error` as a second return argument and return `false, err`.

Reply:
Good spot it should fail.